fdbclient/BulkDumping.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * BulkDumping.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbclient/BulkDumping.h"
+
+BulkDumpState newBulkDumpTaskLocalSST(const KeyRange& range, const std::string& remoteRoot) {
```

Comment:
Do we have to have 'local' vs s3? Could we not distingush by looking at the path/url (perhaps in a follow-on patch?)

Reply:
At this time, I would like to "hardcode" this part for the clarity. However, I will keep in mind that if this is not necessary, I will remove the funcation.

Reply:
Makes sense